tox.ini

```diff
@@ -4,17 +4,12 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26,py27,cover,lint
+envlist = py26,py27,lint
 
 [testenv]
 commands =
     python setup.py dev
-    python setup.py test -q # -q does not suppress errors
-
-[testenv:cover]
-commands =
-    python setup.py dev
-    python setup.py nosetests --with-coverage --cover-min-percentage=47
+    py.test --cov letsencrypt
```

Comment:
`pytest --cov-min` is available only in yet unreleased 2.0 branch: https://github.com/schlamar/pytest-cov/commit/8f679db735ab99f66808e79d4e401996a9191ec0